docs/docs/cli-reference/web.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
sorry; could you add `beta` here too. thanks :D I would have done it, but I am not near my laptop atm.